tests/test_fdjump.py

```diff
@@ -0,0 +1,104 @@
+from pinttestdata import datadir
```

Comment:
Do you do any explicit tests where you set an FDJUMP and just check whether the residuals for the selected TOAs change while the others do not?  I see tests of overall fits but nothing that specific.

Reply:
Added a new test (test_residual_change).